components/services/markdown/src/index.js

```diff
@@ -0,0 +1,39 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+const STATUS_OK = 200
+
+class ServiceMarkdown extends Component {
+  state = {html: ''}
+  async componentDidMount() {
+    const marked = await import('marked')
+    const req = new window.XMLHttpRequest()
+    req.open('GET', this.props.src, false)
+    req.send(null)
+    if (req.status === STATUS_OK) {
+      this.setState({html: marked(req.responseText)})
+    }
+  }
+  render() {
+    const {html} = this.state
+    return (
+      <div className="sui-ServiceMarkdown">
+        <div
+          className="sui-ServiceMarkdown-body"
```

Comment:
maybe you could remove the classnames as you're not using them.

Reply:
How is supposed that we are going to handle with the MD styling on our pages? I mean, for example in MA the static pages have kind of this aspect.

![image](https://user-images.githubusercontent.com/5639972/41297496-88bd14f2-6e5f-11e8-8057-bcd6aa5acbc7.png)

Who will be the responsible to define the different styles of the MD format. The own web-app? a wrapper component on the site studio?